src/clj_async_profiler/ui.clj

```diff
@@ -107,4 +107,7 @@ a { text-decoration: none; }
   [host port dir]
   (stop-server)
   (println "Starting server on port" port)
-  (reset! current-server (wwws/start-server #(handler % dir) host port)))
+  (doto (->> (wwws/start-server #(handler % dir) host port)
```

Comment:
`->>` is better left for operations on collections; it's not idiomatic otherwise. I suggest lifting the server object into a `let` here and remove the threading macros.

Reply:
I use various threading macros (and stuff like doto) all the time to avoid naming things but can refactor for sure.